src/Infrastructure/Reader/TaskYamlReader.php

```diff
@@ -75,6 +71,22 @@ public function readFiles(): ManifestCollectionDto
         return $collection;
     }
 
+    /**
+     * @throws \SprykerSdk\Sdk\Core\Application\Exception\MissingSettingException
+     *
+     * @return \SprykerSdk\SdkContracts\Entity\SettingInterface
+     */
+    protected function getExtensionDirsSetting(): SettingInterface
+    {
+        $taskDirSetting = $this->settingRepository->findOneByPath(Setting::PATH_EXTENSION_DIRS);
```

Comment:
you can use get method. it always present and remove exception

Reply:
Fixed